Kernel/VM/MemoryManager.cpp

```diff
@@ -47,7 +47,8 @@ void MemoryManager::populate_page_directory(PageDirectory& page_directory)
 void MemoryManager::initialize_paging()
 {
     memset(m_page_table_zero, 0, PAGE_SIZE);
-    memset(m_page_table_one, 0, PAGE_SIZE);
+    memset(m_page_table_768, 0, PAGE_SIZE);
+    memset(m_page_table_769, 0, PAGE_SIZE);
```

Comment:
It's interesting that we're clearing the page tables we're currently executing through here.

Reply:
These are separate page tables. The ones used in the `boot.S` are the `boot_page` ones. Now that I think about it, it's probably a waste of time doing this twice.